lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -58,12 +68,18 @@ export function massageMarkdownLinks(content: string): string {
   try {
     const rightSpaces = content.replace(content.trimEnd(), '');
     const matches: UrlMatch[] = [];
-    remark().use(collectLinkPosition(content, matches)).processSync(content);
+    const newContent = content.replace(
+      regEx(/@&#8203;(?<name>[a-zA-Z1-9-!$%^&*()_+|~=`{}[\]:";'<>?,.]*)/gi),
+      '[`@$<name>`](https://togithub.com/$<name>)'
```

Comment:
@HonkingGoose do we still need the code trick when we manually link the user?

Reply:
I don't know. Let's try removing the `code` trick, and see what happens then.

```suggestion
      '[@$<name>](https://togithub.com/$<name>)'
```